src/certificateForm/FormCertificate.js

```diff
@@ -12,21 +12,36 @@ const StyledForm = styled.form`
   grid-template-columns: 1fr 3fr;
 `
 
-export default function FormCertificate({ onFormSubmit, history }) {
-  const [certificate, setCertificate] = useState({
-    date: getISODate(new Date()),
-  })
+export default function FormCertificate({
+  onFormSubmit,
+  editCertificate,
+  history,
+}) {
+  const [certificate, setCertificate] = useState(
+    editCertificate || getNewCertificate()
+  )
+
+  function getNewCertificate() {
+    return {
+      date: getISODate(new Date()),
+      id: uid(),
+      title: '',
+      subject: '',
+      comment: '',
+    }
+  }
 
   function handleInputOnChange(key, value) {
     const param = {}
     param[key] = value
-    console.log('all: ', { ...certificate, ...param })
     setCertificate({ ...certificate, ...param })
   }
 
   function handleOnSubmit(event) {
+    const newEntry = certificate
     event.preventDefault()
-    onFormSubmit(certificate, history)
+    setCertificate(getNewCertificate())
+    onFormSubmit(newEntry, history)
```

Comment:
ausprobieren, ob andere Reihenfolge auch geht, wäre logischer in der Lesart. Noch besser, onFormSubmit auftrennen in Speichern und Navigation und dann

Speichern
Neues Zertifikat
Navigation